lootbox/api.py

```diff
@@ -67,6 +86,14 @@ async def ping_handler() -> data.PingResponse:
     return data.PingResponse(status="ok")
 
 
+@app.get("/time", response_model=int)
```

Comment:
@Andrei-Dolgolev : We should use same structure for this endpoint as Moonstream API - https://github.com/bugout-dev/moonstream/blob/main/backend/moonstreamapi/api.py#L97